implementations/py/schema.graphql

```diff
@@ -0,0 +1,4 @@
+#import { Module } into Http from "ens/wraps.eth:http@1.1.0"
```

Comment:
As we discussed previously, this sort of "import interface into namespace + use the graphql `implements` keyword" is actually "interface extension" and not "implementation" as we want.

I'd recommend doing an un-namespaced import like so:
`#import * from "ens/wraps.eth:http@1.1.0"`

Reply:
I was working on this but it's making CI red since it will require minor changes python bindings